openmetadata-ui/src/main/resources/ui/src/components/Auth/AppAuthenticators/BasicAuthAuthenticator.tsx

```diff
@@ -57,19 +54,15 @@ const BasicAuthenticator = forwardRef(
           );
         }
 
-        if (!refreshToken) {
-          return Promise.reject(new Error(t('message.no-token-available')));
-        }
-
-        const response = await getAccessTokenOnExpiry({
-          refreshToken: refreshToken as string,
-        });
+        const response = await getAccessTokenOnExpiry();
 
-        await setRefreshToken(response.refreshToken);
+        if (response.refreshToken) {
+          await setRefreshToken(response.refreshToken);
+        }
```

Comment:
I think we won't be getting refreshToken anymore cc: @aji-aju

Reply:
addressed, removed